composer.json

```diff
@@ -11,15 +11,7 @@
     ],
     "require": {
         "php": ">=5.4",
-        "squizlabs/php_codesniffer": ">3.0"
+        "squizlabs/php_codesniffer": "^3.0.2",
+        "dealerdirect/phpcodesniffer-composer-installer": "^0.4.1 || ^0.5 || ^0.6"
     },
```

Comment:
keeping this line with a `},` instead of `}` gives me the following error:
```
  [Seld\JsonLint\ParsingException]
  "./composer.json" does not contain valid JSON
  Parse error on line 16:
}
  --------------------^
  Expected: 'STRING' - It appears you have an extra trailing comma
```

I could just remove it myself.. but this time I'm going to wait for your answer before committing/pushing 😃

Reply:
My bad, fixed now (force pushed).